src/main/java/lux/index/IndexConfiguration.java

```diff
@@ -33,7 +33,7 @@
  */
 public class IndexConfiguration {
 
-    public static final Version LUCENE_VERSION = Version.LUCENE_46;
+    public static final Version LUCENE_VERSION = Version.LUCENE_4_10_4;
```

Comment:
I don't think this version needs to be updated.  That version is related to Lucene backwards compatibility.  Lux does not need tokenizers and analyzers that require Lucene features added after 4.6.

https://lucene.apache.org/core/4_7_0/analyzers-common/org/apache/lucene/analysis/core/WhitespaceTokenizer.html#version